<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NumericDataTable</class>
 <widget class="QWidget" name="NumericDataTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="ParameterLabel">
       <property name="text">
        <string>Parameter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="ParameterComboBox"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Control::EventTableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <resources/>
	<customwidgets>
		<customwidget>
			<class>Control::EventTableView</class>
			<extends>QTableView</extends>
			<header>ui/itemviews/event_table_view.h</header>
			<container>1</container>
		</customwidget>
	</customwidgets>
 <connections/>
</ui>
